ingestion/src/main/java/feast/store/serving/redis/RedisStandaloneIngestionClient.java

```diff
@@ -78,8 +78,11 @@ public boolean isConnected() {
   public void sync() {
     // Wait for some time for futures to complete
     // TODO: should this be configurable?
-    LettuceFutures.awaitAll(60, TimeUnit.SECONDS, futures.toArray(new RedisFuture[0]));
-    futures.clear();
+    try {
+      LettuceFutures.awaitAll(60, TimeUnit.SECONDS, futures.toArray(new RedisFuture[0]));
```

Comment:
Should there be a catch?

Reply:
We shouldn't catch here. It will throw RedisException is the case of any command future holds the exception. One of the examples is CommandTimeOutException. It should be thrown so that retry can happen. 